doc/supported_scripts.md

```diff
@@ -95,11 +96,13 @@ third-party library.
 |  [tatar-kryashen_cyrillic](../scriptshifter/tables/data/tatar-kryashen_cyrillic.yml)  |  Tatar-Kryashen (Cyrillic)  |  Y  |  Y  |  stable  |  
 |  [tatar_cyrillic](../scriptshifter/tables/data/tatar_cyrillic.yml)  |  Tatar (Cyrillic)  |  Y  |  Y  |  stable  |  
 |  [telugu](../scriptshifter/tables/data/telugu.yml)  |  Telugu  |  Y  |  Y  |    |  s-to-r lacks capitalization
-|  [thai](../scriptshifter/tables/data/thai.yml)  |  Thai  |  Y  |  Y  |    |  
+|  [thai](../scriptshifter/tables/data/thai.yml)  |  Thai  |  N  |  Y  |  beta  |  
+|  [thai_aksharamukha](../scriptshifter/tables/data/thai_aksharamukha.yml)  |  Thai (Aksharamukha)  |  Y  |  Y  |    |  For Aksharamukha testing
```

Comment:
Thai (Aksharamukha) exists as a configuration but it is not in the index, should we remove it from here too?

Reply:
```suggestion
```